### PR TITLE
ADD v2 API Blueprint. First draft version

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

**Comment by flopezag** [comment]
Just a comment, should not each measurement have a timestamp in which we take it?

**Comment by fgalan** [comment]
@flopezag maybe do you refer to creDate/modDate shown in https://github.com/telefonicaid/fiware-orion/issues/876#issuecomment-108447973? 

This is advanced functionality in v2 that we haven't discussed in detail. Thus, I think that this first version of the v2 apiary doesn't need to include that.

**Comment by fgalan** [comment]
LGTM

File: doc/apiary/v2/fiware-ngsi-simple.apib [hunk 1]
```diff
@@ -0,0 +1,828 @@
...
+                "attributes": [
+                    "temperature"
+                ],
+                "callback": "http://localhost:1234",
+                "options": {
+                    "duration": "1M",
+                    "condition": "ONCHANGE",
+                    "throttling": "PT5S"
+                }
+            },
```
Comment (kzangeli) [inline doc/apiary/v2/fiware-ngsi-simple.apib]: Isn't something missing here?
The attributes to provoke the notification ...
I imagine that

    "attributes": [ "temperature" ],

corresponds to the list of attributes that will be a part of the notification.
I'm missing the other 'list of attributes'.

[ perhaps we changed this funny v1 behaviour for /v2 ... ? ]

Comment (fgalan) [inline doc/apiary/v2/fiware-ngsi-simple.apib]: @kzangeli is right... the triggering attributes list is missing.

However, taking into account that this PR is just to include the draft into the repository, maybe we should pay too much attention to all the details now (as they could be many of them and some depends on the ongoing discussion on v2) and try to fix the details (in sucesive PRs) as we progress in the skype discussion.

Comment (jmcanterafonseca) [inline doc/apiary/v2/fiware-ngsi-simple.apib]: oh yes, well spotted @kzangeli . Landing as per Fermin's comment. and working towards a subsequent version that fixes that issue

**Pull request merged**